pkg/rancher-components/src/components/Card/Card.test.ts

```diff
@@ -0,0 +1,34 @@
+import { mount } from '@vue/test-utils';
+import { Card } from './index';
+
+describe('component: Card', () => {
```

Comment:
It's not a case for the purpose of this tests and we may skip it, however you should always generate a new `wrapper` for each test, while here is the same object shared in more tests.
Keep it in mind if you would have to do some interactions with this component.
To avoid issues in the future, you could wrap all these 3 cases at least into a `describe('given named slots')`.

In an optimal configuration you would also not need to mock the slots for other cases, as well for the card action.